editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/document_node_types.rs

```diff
@@ -1913,6 +1913,51 @@ fn static_nodes() -> Vec<DocumentNodeType> {
 			properties: node_properties::no_properties,
 			..Default::default()
 		},
+		DocumentNodeType {
+			name: "OR",
+			category: "Logic",
+			identifier: NodeImplementation::proto("graphene_core::logic::LogicOrNode<_>"),
+			inputs: vec![
+				DocumentInputType::value("First", TaggedValue::Bool(false), true),
+				DocumentInputType::value("Second", TaggedValue::Bool(false), true),
+			],
+			outputs: vec![DocumentOutputType::new("Output", FrontendGraphDataType::Boolean)],
+			properties: node_properties::logic_operator_properties,
+			..Default::default()
+		},
+		DocumentNodeType {
+			name: "AND",
+			category: "Logic",
+			identifier: NodeImplementation::proto("graphene_core::logic::LogicAndNode<_>"),
+			inputs: vec![
+				DocumentInputType::value("First", TaggedValue::Bool(false), true),
+				DocumentInputType::value("Second", TaggedValue::Bool(false), true),
```

Comment:
We usually set the default exposed values of the second inputs to false (so the last boolean in the `DocumentInputType::value`). This means that they do not by default show up as dots in the node graph leading to reduced clutter. We probably should document this somewhere.

Reply:
Sure, but wouldn't you want to get both inputs to these nodes from other nodes? Most of these nodes either do nothing or always output the same value if you manually set the second value, so I thought that the user would expose the second input manually anyway.

Reply:
Yes - that makes sense.